tests/runtests.py

```diff
@@ -186,8 +186,9 @@ def no_available_apps(self):
     settings.SILENCED_SYSTEM_CHECKS = [
         'fields.W342',  # ForeignKey(unique=True) -> OneToOneField
         'fields.W903',  # NullBooleanField deprecated.
+        'fields.E101',  # BigAutoField migration
```

Comment:
Do we need to silenced it globally? :thinking:

Reply:
I think so - there are certainly a lot of apps inside the tests directory that don't set the PK setting. I'll see if I can remove this.

Reply:
I pushed edits, it shouldn't be necessary anymore.